package.json

```diff
@@ -4,7 +4,7 @@
 	"version": "2.0.0-next.5",
 	"repository": {
 		"type": "git",
-		"url": "git://github.com/browserify/resolve.git"
+		"url": "https://github.com/browserify/resolve.git"
```

Comment:
git+ssh instead please

Reply:
Now changed to `ssh://` per https://github.com/inspect-js/has-symbols/pull/22#discussion_r2274665816